garak/detectors/packagehallucination.py

```diff
@@ -143,3 +143,57 @@ def detect(self, attempt: Attempt) -> List[float]:
             scores.append(1.0 if hallucinated_gem else 0.0)
 
         return scores
+
+
+class JavaScriptNpm(Detector):
+    """Check if the output tries to import or require an npm package not listed in the npm registry"""
+
+    bcp47 = "en"
```

Comment:
Is this for English only? Doesn't the detector only scan JavaScript? If so, `bcp47` should be `*`

Reply:
Updated